packages/babel-plugin-proposal-class-properties/test/fixtures/private/1-helpermemberexpressionfunction/input.js

```diff
@@ -0,0 +1,6 @@
+class D {
+  #arr;
+  f() {
+    for (const el of this.#arr);
```

Comment:
These should be reversed.

Please also add a test for for-in.

Reply:
It is a regression check on the first draft implementation: https://github.com/babel/babel/pull/11703/files/d4c0a5e8b75306f918cf2a2307e4f65130f9ea05#diff-decaf745088978f783d453709a8a8479

Reply:
Ooo, ok. So we need 2 new test cases (for-of and for-in) with it appearing in the `left`.

Reply:
👍 can just do it in the next line, or leave a comment that it shouldn't be transformed

Reply:
personally I think just doing them all in the same class is fine

```js
for (const el of this.#arr);
for (this.#arr of [1, 2]);
for (this.#arr in [1,2,3]);
```

the empty arr seems to be the same case?

and I would rename `1-helpermemberexpressionfunction` to something like for-of-member-expression or something

Reply:
for sure, will remember for next time!